tools/internal_ci/helper_scripts/prepare_ccache_rc

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2022 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Source this rc script to configure ccache (compiler cache to speed up builds)
+
+# Allow use of ccache in builds that support that.
+export GRPC_BUILD_ENABLE_CCACHE=true
+
+# Redis instance housed in grpc-testing cloud project serves as the main compiler cache
+export CCACHE_SECONDARY_STORAGE=redis://10.76.145.84:6379
```

Comment:
We probably want to have a script to rebuild it and doc to describe it. So, people can help to debug if needed.